homeassistant/components/climate/__init__.py

```diff
@@ -51,6 +51,19 @@
 STATE_HEAT_PUMP = 'heat_pump'
 STATE_GAS = 'gas'
 
+SUPPORT_TARGET_TEMPERATURE = 1 << 0
+SUPPORT_TARGET_TEMPERATURE_HIGH = 1 << 1
+SUPPORT_TARGET_TEMPERATURE_LOW = 1 << 2
+SUPPORT_TARGET_HUMIDITY = 1 << 3
+SUPPORT_TARGET_HUMIDITY_HIGH = 1 << 4
+SUPPORT_TARGET_HUMIDITY_LOW = 1 << 5
+SUPPORT_FAN_MODE = 1 << 6
+SUPPORT_OPERATION_MODE = 1 << 7
+SUPPORT_HOLD_MODE = 1 << 8
+SUPPORT_SWING_MODE = 1 << 9
+SUPPORT_AWAY_MODE = 1 << 10
+SUPPORT_AUX_HEAT = 1 << 11
```

Comment:
I think for 90% of users is that to complex :) Use integer like other components with supported features